tests/end-to-end/api/00-miscellaneous.js

```diff
@@ -228,7 +228,6 @@ describe('miscellaneous', function() {
 					expect(res.body).to.have.property('offset');
 					expect(res.body).to.have.property('total');
 					expect(res.body).to.have.property('count');
-					expect(res.body.result[0]).to.have.property('_id');
```

Comment:
I'm not sure but are our changes affecting these test cases? I thought we just added additional endpoints.